Task2_1/PointWithMass.cs

```diff
@@ -0,0 +1,52 @@
+﻿
+namespace Task2_1
+{
+    internal class PointWithMass
+    {
+        private int[] _coordinates = new int[3];
+        private double _mass;
+
+        public PointWithMass(int x, int y, int z, double mass)
+        {
+            X = x;
+            Y = y;
+            Z = z;
```

Comment:
No need to check it here, you use property Mass, this activity will be done there (Line 39)

Reply:
Good Morning @IgorKheidorov 

I completely forgot to change that line haha.

Changes implemented. Proceeded to merge.